errortypes/errortypes.go

```diff
@@ -165,6 +167,31 @@ func (err *BidderTemporarilyDisabled) Severity() Severity {
 	return SeverityWarning
 }
 
+// NoConversionRate is thrown by the currency.Conversions GetRate(from string, to string) method
+// when the conversion rate between the two currencies, nor its reciprocal, can be found.
+type NoConversionRate struct {
+	fromCur, toCur string
+}
+
+func NewNoConversionRateError(fromCur, toCur string) *NoConversionRate {
+	return &NoConversionRate{
+		fromCur: fromCur,
+		toCur:   toCur,
+	}
+}
+
+func (err *NoConversionRate) Error() string {
+	return fmt.Sprintf("Currency conversion rate not found: '%s' => '%s'", err.fromCur, err.toCur)
+}
+
+func (err *NoConversionRate) Code() int {
+	return NoConversionRateErrorCode
+}
+
+func (err *NoConversionRate) Severity() Severity {
+	return SeverityUnknown
```

Comment:
Please choose a severity of either Fatal (the auction is stopped when the error is encountered) or Warning (the auction continues and a warning is written to the debug section of the response). A vanilla Go error is considered fatal, so that might make the most sense here - especially since you've defined the code as an error code as opposed to a warning code.

Reply:
The error gets ignored in `exchange/bidder.go` so the severity doesn't really matter: